StripeFinancialConnections/StripeFinancialConnections/Source/Native/LinkAccountPicker/LinkAccountPickerViewController.swift

```diff
@@ -229,7 +229,11 @@ final class LinkAccountPickerViewController: UIViewController {
         self.footerView = footerView
         footerContainerView.addAndPinSubview(footerView)
 
-        bodyView.selectAccounts([]) // activate the logic to list all accounts
+        let firstSelectableAccount = accountTuples.first { accountTuple in
+            accountTuple.accountPickerAccount.allowSelection && accountTuple.partnerAccount.allowSelectionNonOptional
+        }
```

Comment:
can we also check that there's no drawer on selection for that account? see [Android](https://github.com/stripe/stripe-android/blob/6c8953e7645b0b4e280fd9bc1a6a31e403683a19/financial-connections/src/main/java/com/stripe/android/financialconnections/features/linkaccountpicker/LinkAccountPickerViewModel.kt#L106)

Reply:
Good shout! `drawerOnSelection` doesn't exist on the partner account model in `master`, so I'll close this branch and move this to https://github.com/stripe/stripe-ios/pull/3935 which points to the NME feature branch